packages/button/src/mwc-button.ts

```diff
@@ -21,7 +21,7 @@ import {style} from './mwc-button-css.js';
 
 @customElement('mwc-button')
 export class Button extends ButtonBase {
-  static styles = style;
+  protected static styles = style;
```

Comment:
I'm seeing a bunch of errors about this. `styles` is public in `LitElement`, so we can't restrict the visibility.

Actually I see the same for a few other methods too, e.g. `disconnectedCallback`, which is public in `HTMLElement`.

Mind running `npm run build` and fixing all the new type errors?

Reply:
Done! https://github.com/material-components/material-components-web-components/pull/371/commits/5c7823beb356478f20f055245eb595375175dc94 and https://github.com/material-components/material-components-web-components/pull/371/commits/f9fb1901d425327b195c3b45718fc8f0ad827ea7